Common/Securities/BuyingPowerModelExtensions.cs

```diff
@@ -96,5 +96,49 @@ OrderDirection direction
             // existing implementations assume certain non-account currency units, so return raw value
             return buyingPower.Value;
         }
+
+        /// <summary>
+        /// Gets the margin currently allocated to the specified holding
+        /// </summary>
+        /// <param name="model">The buying power model</param>
+        /// <param name="security">The security</param>
+        /// <returns>The maintenance margin required for the </returns>
```

Comment:
nit - `The maintenance margin required for the `... `current security holdings`?